ext/websocket/lib.rs

```diff
@@ -567,6 +570,94 @@ pub fn op_ws_get_error(state: &mut OpState, rid: ResourceId) -> String {
   resource.error.take().unwrap_or_default()
 }
 
+#[op(v8)]
+pub fn op_ws_loop(
+  scope: &mut v8::HandleScope,
+  state: Rc<RefCell<OpState>>,
+  rid: ResourceId,
+  callback: serde_v8::Value,
+) -> Result<(), AnyError> {
+  let cb = event::JsCb::new(scope, callback);
+
+  let resource = state
+    .borrow_mut()
+    .resource_table
+    .get::<ServerWebSocket>(rid)?;
+
+  // SAFETY: Future runs on the main thread.
+  tokio::task::spawn(unsafe {
+    MaskFutureAsSend::new(async move {
```

Comment:
Can we not use `deno_core::tasks::spawn`?

Reply:
We can and should